src/codeu/chat/DerbyStore.java

```diff
@@ -122,6 +168,7 @@ public void addUser(User u) throws SQLException {
 		addUser.executeUpdate();
 	}
 	
+	@Override
 	public void addMessage(Message m) throws SQLException {
 		PreparedStatement addMessage = conn.prepareStatement(addMessageInfo);
 		addMessage.setString(1, removeCharsInUuid(m.id.toString()));
```

Comment:
Like above, are there any uniqueness checks that should be added here? What if two messages have the same ID?